### ISSUE TITLE
How to run training if input is given using a generator?

### ISSUE THREAD

**Issue by sohamsatyadharma** [4uiiurz1/keras-arcface#31]
If the input is given as 
`model = Model([input, label], output)`
and a generator object is used to call `model.fit` like
`model.fit(input_generator, ....)`

How will the labels be passed to the fit function?
